CHANGELOG.md

```diff
@@ -11,12 +11,12 @@ Changelog
 
 - `pyquil.version.__version__` has been moved to `pyquil.__version__`.
 
-- `PyquilConfig` has been replaced by `QCSClientConfiguration`. As a result, the only supported configuration-related
+- `PyquilConfig` has been replaced by `api.QCSClientConfiguration`. As a result, the only supported configuration-related
    environment variables are:
     - `QCS_SETTINGS_APPLICATIONS_PYQUIL_QVM_URL` (replaces `QVM_URL`), and
     - `QCS_SETTINGS_APPLICATIONS_PYQUIL_QUILC_URL` (replaces `QUILC_URL`)
 
-- `ForestConnection` and `ForestSession` have been replaced by `api.Client`.
+- `ForestConnection` and `ForestSession` have been replaced by `api.EngagementManager`.
```

Comment:
Only partially replaced though, right? The ForestSession was a `requests.session` and so is somewhat replaced by the `qcs_client`

Reply:
True. I'll come up with a clearer sentence.

Reply:
OK @kalzoo, let me know if the new phrasing is any better.